cli/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -73,6 +74,7 @@ pub const FLOX_ENV_DIRS_VAR: &str = "FLOX_ENV_DIRS";
 pub const FLOX_ENV_LIB_DIRS_VAR: &str = "FLOX_ENV_LIB_DIRS";
 pub const FLOX_ACTIVE_ENVIRONMENTS_VAR: &str = "_FLOX_ACTIVE_ENVIRONMENTS";
 pub const FLOX_PROMPT_ENVIRONMENTS_VAR: &str = "FLOX_PROMPT_ENVIRONMENTS";
+pub const FLOX_PROCESS_COMPOSE_SOCKET_VAR: &str = "_FLOX_PC_SOCKET";
```

Comment:
The abbreviated `PC` is a bit ambiguous. I was already in two minds about whether we should generalise these names to `[FLOX_]SERVICES_SOCKET_[NAME|VAR]` and `services.sock` so that we don't leak too much information about the implementation and make it easier if we ever need to change in the future (albeit with some dancing). Not needing to abbreviate it seems like another good reason.
```suggestion
pub const FLOX_SERVICES_SOCKET_VAR: &str = "_FLOX_SERVICES_SOCKET";
```